apps/omg_conformance/test/omg_conformance/conformance/merkle_proof_test.exs

```diff
@@ -32,12 +32,16 @@ defmodule OMG.Conformance.MerkleProofTest do
   @max_block_size trunc(:math.pow(2, @proof_length))
 
   setup_all do
+    :ok = Application.put_env(:ex_plasma, :exit_id_size, 160)
```

Comment:
why these tests don't work with `160` size?

Reply:
some actually do, but I didn't want to dive into this in this PR, but in a follow up!